explainer.md

```diff
@@ -499,6 +512,8 @@ interface VRDevice : EventTarget {
 
   Promise<boolean> supportsSession(VRSessionCreateParametersInit parameters);
   Promise<VRSession> requestSession(VRSessionCreateParametersInit parameters);
+
+  Promise<void> ensureContextCompatibility(WebGLRenderingContext context);
```

Comment:
Agree with this, or perhaps use a union type if you want to refer to a data type that's exposed to JavaScript. (WebGLRenderingContextBase isn't.)